templates/master.yaml.tmpl

```diff
@@ -38,34 +38,30 @@ storage:
         id: 0
       contents:
         source: "data:text/plain;charset=utf-8;base64,{{ index .Files "k8s-resource/aws-cni.yaml" }}"
-    - path: /srv/calico-policy-only.yaml
-      filesystem: root
-      mode: 420
-      user:
-        id: 0
-      group:
-        id: 0
-      contents:
-        source: "data:text/plain;charset=utf-8;base64,{{ index .Files "k8s-resource/calico-policy-only.yaml" }}"
-    - path: /srv/calico-crd-installer.yaml
+{{ end }}
+{{ if eq .Provider "azure" }}
+    - path: /opt/azure-cni
       filesystem: root
-      mode: 420
+      mode: 356
       user:
         id: 0
       group:
         id: 0
       contents:
-        source: "data:text/plain;charset=utf-8;base64,{{ index .Files "k8s-resource/calico-crd-installer.yaml" }}"
-    - path: /srv/calico-crd-installer-rbac.yaml
+        source: "data:text/plain;charset=utf-8;base64,{{ index .Files "conf/azure-cni" }}"
+{{ end }}
+{{ if or (eq .Provider "aws") (eq .Provider "azure") }}
+    - path: /srv/calico-policy-only.yaml
       filesystem: root
       mode: 420
       user:
         id: 0
       group:
         id: 0
       contents:
-        source: "data:text/plain;charset=utf-8;base64,{{ index .Files "k8s-resource/calico-crd-installer-rbac.yaml" }}"
-{{ else }}
+        source: "data:text/plain;charset=utf-8;base64,{{ index .Files "k8s-resource/calico-policy-only.yaml" }}"
+{{ end }}
+{{ if ne .Provider "aws" }}
     - path: /srv/calico-all.yaml
```

Comment:
Still need this manifest on azure because we need to cleanup old resources before we switch to policy-only